gpytorch/lazy/lazy_tensor.py

```diff
@@ -731,6 +731,106 @@ def add_jitter(self, jitter_val=1e-3):
         diag = torch.tensor(jitter_val, dtype=self.dtype, device=self.device)
         return self.add_diag(diag)
 
+    def cat_rows(self, fant_train, fant_fant):
+        """
+        Concatenates new rows and columns to the matrix that this LazyTensor represents, e.g.
+        C = ((A; B); B', D).
```

Comment:
... where `A` is the existing lazy tensor and `B` and `D` are the new components...